code/modules/power/fusion/fusion_circuits.dm

```diff
@@ -9,23 +9,25 @@
 	board_type = "machine"
 	origin_tech = "{'programming':4,'engineering':4,'materials':4}"
 	req_components = list(
+		/obj/item/stock_parts/keyboard = 1,
```

Comment:
IIRC keyobard/screen components should be in additional_spawn_components rather than in req ones, or entire machine will stop working if screen is broken

Reply:
Seems inconsistent in code, base circuitboards for example have them in `req_components`.

Reply:
`[10:07 AM] afterthought: they should be in req_components if you want the machine to not function without them, iirc`

Reply:
I'm ok with the machine breaking if they're broken since it has no other use.